Cake.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+final class Cake
+{
+    public function __construct(private readonly int $candles)
+    {
+    }
+
+    public function serve(): void
+    {
+        if ($this->candles > 0) {
```

Comment:
looks like this logic is inverted.. we want the error to occur when there are no candles, right?